azure-ts-appservice-springboot/infrastructure/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "azure-typescript",
+    "devDependencies": {
+        "@types/node": "latest"
+    },
+    "dependencies": {
+        "@pulumi/azure": "^0.19.5",
+        "@pulumi/docker": "^0.17.2",
+        "@pulumi/pulumi": "^0.17.28"
```

Comment:
I think most examples depend on `latest` package versions to prevent the need to update them.

Reply:
Done